### PR TITLE
fix(file-uploader): Make over max file size errors more verbose

### PR DESCRIPTION
**Pull request by ErikCH**
#### Description of changes

The max file size error message, is short, and confusing. This PR adds more information to the max file size errors. 

Also in smaller screens the message gets cut off. So now it's on its own line. 

#### Issue #, if available

[Asana](https://app.asana.com/0/1202067479136208/1203495483940613/f)

Before:
image

Updated text:
image

#### Description of how you validated changes
Updated snapshots

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are updated
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react/src/components/Storage/FileUploader/UploadTracker/index.tsx [hunk 1]
```diff
@@ -153,51 +148,58 @@ export function UploadTracker({
...
+            isDeterminate={isDeterminate}
+            isPercentageTextHidden
+          />
+        ) : null}
+      </View>
+      <UploadMessage
+        fileState={fileState}
+        errorMessage={errorMessage}
+        percentage={percentage}
+      />
```
Comment (ErikCH) [inline packages/react/src/components/Storage/FileUploader/UploadTracker/index.tsx:202]: The biggest change was moving this to the bottom so it's on it's own line

File: packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts [hunk 2]
```diff
@@ -75,7 +75,9 @@ export function humanFileSize(bytes, si = false, dp = 1) {
 export const checkMaxSize = (maxSize: number, file: File): string | null => {
   if (!maxSize) return null;
   if (file.size > maxSize) {
-    return translate('Above max ') + humanFileSize(maxSize, true);
+    return (
+      translate('Above file size limit of ') + humanFileSize(maxSize, true)
+    );
```
Comment (ErikCH) [inline packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts:80]: New verbose text

File: packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts [hunk 3]
```diff
@@ -75,7 +75,9 @@ export function humanFileSize(bytes, si = false, dp = 1) {
 export const checkMaxSize = (maxSize: number, file: File): string | null => {
   if (!maxSize) return null;
   if (file.size > maxSize) {
-    return translate('Above max ') + humanFileSize(maxSize, true);
+    return (
+      translate('Above file size limit of ') + humanFileSize(maxSize, true)
```
Comment (reesscot) [inline packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts:79]: Starting with "Above" still reads as confusing to me, who about this?
```suggestion
      translate('File size must be below ') + humanFileSize(maxSize, true)
```

Comment (ErikCH) [reply packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts:79]: I like it, I'll change it

Comment (ErikCH) [reply packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts]: [e1371c8](https://github.com/aws-amplify/amplify-ui/pull/3284/commits/e1371c81b8cc46bdb5e69a5ad606804e8949c544)

**Review by reesscot — APPROVED** [review]

**Review by wlee221 — APPROVED** [review]

**Pull request merged**